apps/io-services-cms-webapp/ServiceReviewChecker/function.json

```diff
@@ -0,0 +1,13 @@
+{
+  "disabled": true,
```

Comment:
Unfortunately, this flag doesn't work (although docs are not clear about it). The safest method to disable the function is to add this to application settings:
```
"AzureWebJobs.ServiceReviewChecker.Disabled" = "1"
```